src/app/timer/timer-format.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'timerFormat',
+})
+export class TimerFormat implements PipeTransform {
+  transform(value: number): string {
```

Comment:
add comment that value is expected to be passed in seconds

Reply:
done